bleak/backends/winrt/client.py

```diff
@@ -442,9 +458,14 @@ async def get_services(self, **kwargs) -> BleakGATTServiceCollection:
             return self.services
         else:
             logger.debug("Get Services...")
+            use_cached_services = kwargs.get(
+                "use_cached_services", self._use_cached_services
+            )
             services: Sequence[GattDeviceService] = _ensure_success(
                 await self._requester.get_gatt_services_async(
-                    BluetoothCacheMode.UNCACHED
+                    BluetoothCacheMode.CACHED
+                    if use_cached_services
+                    else BluetoothCacheMode.UNCACHED
```

Comment:
there would be less duplicate code if we just did this conversion from `bool` to `BluetoothCacheMode` once above instead of in each function call.

Reply:
Yes. Good call. This was lazy coding.